probes/hasDangerousWorkflowScriptInjection/patch/testdata/twoInjectionsDifferentJobs.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+on:
+	issue:
+
+jobs:
+	fascinating-job:
+		steps:
```

Comment:
I notice your synthetic examples all use tabs for indentation. That's a good test to make sure the remediation keeps that syntax (it currently doesn't, in this case!), but note that the vast majority of real workflows use 2-space indentation instead.

Reply:
Hmm nice point. I'll change the indentation of my tests to use 2-space, to follow this pattern -- but on the test about Indentation, I'll mix the 2 types of indentation and also test this out hehe

Reply:
FWI, by [this documentation](https://learnxinyminutes.com/docs/yaml/)  and this [YAML validator](https://www.yamllint.com), tab-indentation is actually not even allowed/recognized on official YAML

Reply:
TIL! That's neat! Also makes a lot of sense (imagine a doc that mixes spaces and tabs... the "structure" of the doc will appear different to people who use tabs = 2 or 4 spaces).